doc/index.rst

```diff
@@ -24,28 +51,43 @@ for testing.
    $DESIMODEL : data used for simulating DESI <DESIMODEL/index>
    $DESISURVEY_OUTPUT : outputs from desisurvey and surveysim <DESISURVEY_OUTPUT/index>
    $PROTODESI : data and logs from the ProtoDESI campaign <PROTODESI/index>
+
+
+Other information
+=================
+
+.. toctree::
+   :maxdepth: 1
+
    bitmasks
-   datamodel
-   api
-   changes
+   Environment variables <envvar>
 
 Imaging data and their catalogs are documented separately by the
 `Legacy Survey <https://www.legacysurvey.org/>`_.
 
 The desidatamodel_ package on GitHub includes the data model input files
 and some Python utility code for generating and checking data model files.
```

Comment:
Add a sentence to this paragraph along the lines of: the links below will be useful for developers.

Reply:
To be clear, the paragraph I mean is the one starting `The desidatamodel_ package`.

Reply:
done